Add KYC sharing reference (#92)
uphold/topper-docs: docs/kyc-shaing.md

---
sidebar_position: 16
---

# KYC sharing

To enhance the user experience we offer a REST API that our partners can use in order to share user KYC processes.

> This will improve the user experience when reaching Topper. As more information we're able to gather from our partners, as less will be requested to the user, smoothing the sign up process.

Reviewer: ```suggestion
This will improve the user experience when reaching Topper. If we can gather more information from our partners, Topper will need to request less information from the user, smoothing the sign-up process.
```
